Directory.Build.props

```diff
@@ -90,11 +90,14 @@
     <RuntimeInstallerBaseName>aspnetcore-runtime</RuntimeInstallerBaseName>
     <TargetingPackInstallerBaseName>aspnetcore-targeting-pack</TargetingPackInstallerBaseName>
 
+    <!-- 3.0.1 is a special servicing release since we're building the targeting pack. We need this condition in a few places. -->
+    <Is301 Condition="'$(VersionPrefix)' == '3.0.1'">true</Is301>
```

Comment:
I'm not fond of this name since this isn't particular specific to 3.0.1. This is applicable when we are building the targeting pack as part of a servicing release. Maybe IsPatchingTargetingPack?

Reply:
You're right, that's better. Will fix